src/piggui.rs

```diff
@@ -37,6 +37,13 @@ pub enum Layout {
 
 impl Layout {
     const ALL: [Layout; 2] = [Layout::BoardLayout, Layout::BCMLayout];
+
+    fn get_spacing(&self) -> u16 {
+        match self {
+            Layout::BoardLayout => 470,
```

Comment:
I wouldn't delay this PR more with this change, but in the future maybe we can introduce constants for each dimension, then they can be added-up (to other constants) to get higher-level dimensions. Then changes in lower level widget sizes automatically change higher-up dimensions and you don't need to work them all out and edit manually.